package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "loopback-context",
-  "version": "3.3.0",
+  "version": "3.4.0",
```

Comment:
there's no need to change the version.  When we release, it will update the version number automatically.

Reply:
ok, changed back the version.